package.json

```diff
@@ -72,11 +72,13 @@
   "devDependencies": {
     "@babel/plugin-proposal-class-properties": "^7.16.7",
     "@babel/plugin-proposal-private-property-in-object": "^7.16.7",
+    "@babel/preset-react": "^7.18.6",
+    "@babel/preset-typescript": "^7.18.6",
     "@commitlint/cli": "^16.2.4",
     "@commitlint/config-conventional": "^16.2.4",
-    "@cypress/react": "^5.12.3",
-    "@cypress/webpack-dev-server": "^1.8.1",
-    "@cypress/webpack-preprocessor": "^5.11.1",
+    "@cypress/react": "^6.1.1",
```

Comment:
@peterslany do we need to keep the Cypress imports?

Reply:
Not really, I will remove the cypress dependencies.